lib/generators/email_events/templates/create_sent_email_data.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSentEmailData < ActiveRecord::Migration
```

Comment:
It's a migration template.  `rails g email_events:install` creates the migration in the app.  This adds a table necessary for tracking sent e-mails.

Note that we already have this table, so the migration there is a custom one to just modify the existing table.